formspree/templates/email/pre_inline_style/90-percent-warning.html

```diff
@@ -33,7 +33,7 @@
 									</tr>
 									<tr>
 										<td class="content-block">
-											We want to let you know that you've approached the 90% of the limit of {{ config.SERVICE_NAME }} monthly limit. Please note that you get up to {{ config.MONTHLY_SUBMISSIONS_LIMIT }} submissions per month as a non {{ config.UPGRADED_PLAN_NAME }} customer, before your form will be disabled.
+											We want to let you know that you've approached the 90% of the limit of {{ config.SERVICE_NAME }} monthly limit. Please note that you get up to {{ limit }} submissions per month as a non {{ config.UPGRADED_PLAN_NAME }} customer, before your form will be disabled.
```

Comment:
You've reached 90% of the {{ config.SERVICE_NAME }} free monthly limit. You get up to {{ limit }} submissions per month for free. After that we will keep up to {{ config.ARCHIVED_SUBMISSIONS_LIMIT }} past submissions in our archive that you can access at any time if you upgrade.